checkers/model_form.py

```diff
@@ -37,24 +42,46 @@ def is_model_form_attribute(self, base):
             base.value.id == 'models' and base.attr == 'ModelForm'
         )
 
+    def is_string_dunder_all(self, element):
+        """
+        Return True if element is ast.Str and equals "__all__"
+        """
+        return (
+            isinstance(element.value, ast.Str) and
```

Comment:
In Python 3, `b'__all__'` and `'__all__'` are represented by two different types of nodes:

```python
>>> ast.parse('b"hello"').body[0].value
<_ast.Bytes at 0x104f4edd8>
>>> ast.parse('"hello"').body[0].value
<_ast.Str at 0x105011c18>
```

So you have to add a check for `ast.Bytes`.

Reply:
good catch 👍